include/admin.jsonnet

```diff
@@ -85,7 +85,16 @@ local common = import "common.jsonnet";
                       {
                         "name": "CONFIGURATION_SERVICE_PORT",
                         "value": "5672"
-                      }]),
+                      },
+                      {
+                        "name": "API_SERVER_HOSTNAME",
```

Comment:
Fair points, and it is only a name so not a huge issue at this point. API_SERVER is a little too ambiguous/generic for my taste (though as an env var I can certainly live with it :). What about MESSAGING_CONTROL or MESSAGING_ADMIN or something a little more descriptive of its actual function?